helm/libcoder/templates/_rbac.yaml

```diff
@@ -61,4 +61,160 @@ roleRef:
   kind: Role
   name: {{ .Values.coder.serviceAccount.name }}-workspace-perms
 {{- end }}
+{{- if .Values.coder.serviceAccount.workspaceNamespaces }}
+{{- if .Values.coder.serviceAccount.workspaceNamespaces.namespaces }}
+{{- range .Values.coder.serviceAccount.workspaceNamespaces.namespaces }}
+{{- if .name }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ $.Values.coder.serviceAccount.name }}-workspace-perms
+  namespace: {{ .name }}
+rules:
+  {{- if (hasKey . "workspacePerms") }}
+    {{- if .workspacePerms }}
```

Comment:
Does a pipeline like `if (.workspacePerms | default $.Values.coder.serviceAccount.workspacePerms)` work instead? Would prevent a lot of duplication